### PR TITLE
refs task[38145] Build product details

### PR DESCRIPTION
**Pull request by haitt-2557**
https://edu-redmine.sun-asterisk.vn/issues/38145

### CODE REVIEW

**Comment by haitt-2557** [comment]
ready

**Comment by haitt-2557** [comment]
ready

File: src/pug/pages/productDetails.pug [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                            .col-lg-12
+                                +titleSection("N","SẢN PHẨM NỔI BẬT","20","21","22")
+                            .col-lg-12.col-md-12
+                                .row
+                                    +carousel(newProducts,3,"carouselProduct20","desktopCarousel",4,6,12)
+                                    +carousel(newProducts,2,"carouselProduct21","laptopCarousel",4,6,12)
+                                    +carousel(newProducts,1,"carouselProduct22","mobileCarousel",4,6,12)   
+
+                                        
+
```
Comment (manhnt-0960) [inline src/pug/pages/productDetails.pug]: xóa dòng trống đi e

**Comment by haitt-2557** [comment]
ready

**Pull request merged**